tests/sqltest/contact.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+FROM v1.couchdb couchdb
```

Comment:
Can we use the values set in the environment variables for the schema and the root table here and in the other tests?